cli.js

```diff
@@ -25,7 +25,8 @@ const cli = meow(`
 	  $ imagemin foo.png > foo-optimized.png
 	  $ cat foo.png | imagemin > foo-optimized.png
 	  $ imagemin foo.png --plugin=pngquant > foo-optimized.png
-	  $ imagemin foo.png --plugin.pngquant.quality={0.1,0.2} > foo-optimized.png
+	  $ imagemin foo.png --plugin.pngquant.quality=0.1 --plugin.pngquant.quality=0.2 > foo-optimized.png
+	  $ imagemin foo.png --plugin.pngquant.quality={0.1,0.2} > foo-optimized.png (short syntax for non-Windows users)
```

Comment:
The comment needs to be above using `#` syntax as the command is meant to be copy-pastable.

You also need to sync it to the readme.